src/components/header.tsx

```diff
@@ -0,0 +1,18 @@
+import Image from 'next/image';
+import profilePicture from "../../public/profile picture.jpg";
```

Comment:
Might be a good idea to rename this to not have a space in it. Spaces in filenames can sometimes have unexpected results. 
```suggestion
import profilePicture from "../../public/profile-picture.jpg";
```

Reply:
Done! Renamed the profile picture file in a following commit. Thanks!